fandogh_cli/domain_commands.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+import click
+
+from .fandogh_client.domain_client import *
+from .base_commands import FandoghCommand
+from .config import *
+from .presenter import present
+from .utils import login_required, format_text, TextStyle
+
+
+@click.group("domain")
+def domain():
+    """
+    Domain management commands
+    """
+    pass
+
+
+@click.command("add", cls=FandoghCommand)
+@click.option('--name', prompt='domain name', help='your domain name')
+@login_required
+def add(name):
```

Comment:
Isn't it easier if we actually verify inside the `add` command?
We call the API to submit new domain, then ask user to set the TXT record and press enter, and then after user pressed enter we verify the domain, if TXT record was there then we display some message and quite other wise we repeat this process until user Ctrl+C

Reply:
1- POST /api/domains
2- Ask user to set TXT record and press Enter
3-1 if TXT record is there quit
3-2 display a warning and go to 2

Reply:
Good idea, will do it in this way

Reply:
done